src/components/PaymentTile.js

```diff
@@ -100,15 +105,42 @@ const PaymentTile = (props) => {
         currencyInformation: currencyInformation
     })
     const numberOfContributorsAllocated = allocations.length
-
-    const projectName = (allocations.length > 0) ? allocations[0].project.name : ''
+    const numberOfFilteredContributorsAllocated = project 
+        ? filteredAllocations.length 
+        : null
     
+    const totalProjectAllocations = () => {
+        let total = 0
+        filteredAllocations.map(f => {
+            total += (f.amount / 100)
+        })
+        return total
+    }
+    const totalAllocatedContributors = project 
+        ? (formatAmount({
+            amount: totalProjectAllocations(),
+            currencyInformation: currencyInformation
+        }))
+        : null
+
+    const calculateAllocationsOtherProjects = () => {
+        if (project) {
+            return dataTotalAllocated.getPaymentById.totalAllocated / 100 - totalProjectAllocations()
+        }
+    }
+    const totalAllocatedOtherProjects = formatAmount({
+        amount: calculateAllocationsOtherProjects(),
+        currencyInformation: currencyInformation
+    })
+
     const renderPaymentAllocations = (props) => {
 
         const {
             allocations,
             currencyInformation
         } = props
+        const projects = []
+        let projectTitle = null
```

Comment:
@Ovifer13 This `projectTitle` should be inside the `allocations.map` iterator so that you have a unique instance of the variable for each render inside the `map` loop. If it's out here it might get overwritten each pass of the loop